src/sentry/options/defaults.py

```diff
@@ -2343,6 +2343,11 @@
     default=120,  # 2 minutes
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "standalone-spans.detect-performance-issues-consumer.enable",
+    default=True,
```

Comment:
default to true because this is more of a killswitch/drain flag. this should be on by default when we enable the pipeline.